RpcNep5Tracker/RpcNep5Tracker/config.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "PluginConfiguration": {
+    "DBPath": "Nep5BalanceData",
+    "TrackHistory" : true
```

Comment:
Should this be `false` by default?

Reply:
I guess it depends whether people want less by default or more by default. My thought was people would want more by default.

Reply:
One other thought is that you could turn it off later and remove the keys from leveldb more easily than turning it on later and having to resync the chain.